game-core/src/main/java/games/strategy/triplea/delegate/UnitComparator.java

```diff
@@ -127,9 +127,9 @@ private UnitComparator() {}
         .thenComparing(getMovableUnitsComparator(units, route));
   }
 
-  static Comparator<Unit> getDecreasingAttackComparator(final PlayerId player) {
+  static Comparator<Unit> getDecreasingBombardComparator() {
     return Comparator.comparing(Unit::getType,
         Comparator.comparing(UnitAttachment::get,
-            Comparator.<UnitAttachment>comparingInt(u -> u.getAttack(player)).reversed()));
+            Comparator.<UnitAttachment>comparingInt(u -> u.getBombard()).reversed()));
```

Comment:
```suggestion
            Comparator.comparingInt(UnitAttachment::getBombard).reversed()));
```
I believe this should work